src/main/java/seedu/address/logic/commands/EditSupplierCommand.java

```diff
@@ -88,6 +91,17 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(String.format(MESSAGE_EDIT_SUPPLIER_SUCCESS, editedSupplier));
     }
 
+    /**
+     * combine the offer set of supplierToEdit with EditSupplierDescriptor
+     *
+     * return the combined set
+     */
```

Comment:
Perhaps you can follow standard javadoc format?

Reply:
this branch only updates the function of edit-s, another branch and pr will be created to update the test cases.